src/fontra/core/pathops.py

```diff
@@ -0,0 +1,54 @@
+import pathops
+from fontTools.pens.pointPen import (
+    GuessSmoothPointPen,
+    PointToSegmentPen,
+    SegmentToPointPen,
+)
+
+from .classes import structure, unstructure
+from .path import PackedPath, PackedPathPointPen
+
+
+def skiaPathOperations(pathA, pathB, pathOperation):
+    fontraPathA = structure(pathA, PackedPath)
```

Comment:
The structure/unstructure parts belong in serverutils.py: it's a consequence of how the web api funcs work, and should not affect this library module.

Reply:
done, see: https://github.com/googlefonts/fontra/pull/1582/commits/2c37dc2d96eee1a6f0fc6fcb71cbcc2497a51d3b